components/ILIAS/Mail/classes/Signature/Placeholder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Mail\Signature;
+
+interface Placeholder
+{
+    public function getId(): string;
+    public function getLabel(): string;
+    public function handle(Signature $signature): ?array;
+    public function setNext(self $next): self;
+    public function getNext(): ?self;
```

Comment:
Please describe the shape of the array for `?array`.

```suggestion
    public function getId(): string;

    public function getLabel(): string;

    public function handle(Signature $signature): ?array;

    public function setNext(self $next): self;

    public function getNext(): ?self;
```